commands/logs_e2e_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mmctl/client"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestlogsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("Display single log line", func(c client.Client) {
+		cmd := &cobra.Command{}
+		cmd.Flags().Int("number", 1, "")
+
+		data, err := testLogsCmdF(c, cmd, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(data, 2)
+	})
+
+	s.RunForAllClients("Display in logrus for formatting", func(c client.Client) {
+		cmd := &cobra.Command{}
+		cmd.Flags().Bool("logrus", true, "")
+		cmd.Flags().Int("number", 1, "")
+
+		data, err := testLogsCmdF(s.th.SystemAdminClient, cmd, []string{})
+		s.Require().Nil(err)
+		s.Require().Len(data, 2)
+		s.Contains(data[1], "level=info msg=\"Syncing plugins from the file store\"")
```

Comment:
I'd go for something slightly more generic gere, like:

```suggestion
		s.Contains(data[1], "time=")
		s.Contains(data[1], "level=")
		s.Contains(data[1], "msg=")
```

Reply:
updated